main.cpp

```diff
@@ -266,6 +329,24 @@ float Restaurant::chargeMoney(float moneyOfMeal)
     return moneyOfMeal;
 }
 
+void Restaurant::cleanTheKitchen(int amountOfAmmonia, int numOfSponges)
+{
+    bool kitchenIsClean = true;
+    int sponges = 0;
+    int ammonia = 0;
+    int cleanTheKitchen = amountOfAmmonia + numOfSponges;
+
+    while(sponges && ammonia <= 1)
+    {
+        ++sponges;
+        ++ammonia;
+        ++cleanTheKitchen;
+        std::cout << "The kitchen is being cleaned!" << std::endl;
+        if(kitchenIsClean)
+        break;
```

Comment:
Indent the body of any `if` blocks, even if they're a single statement/single line
```
if( condition )
    doSomething();
```
not:
```
if( condition )
doSomething();
```
- [ ] revise